### ISSUE TITLE
Add option to disable sanitation for column names.

### ISSUE THREAD

**Issue by ilyanoskov** [awslabs/aws-data-wrangler#278]
**Is your idea related to a problem? Please describe.**
Currently I am forced to avoid .to_csv method if I need specific column names (that have spaces in the name for example)

**Describe the solution you'd like**
It would be great to use .to_csv(sanitize_columns=False) option to keep original column names.

**Comment by igorborgest** [comment]
Thanks @ilyanoskov !

I've just fixed it in our development branch, and will be released soon on version `1.5.0`.

Now, we have the suggested argument `sanitize_columns`.

Could you test it, and give us some feedback?

To install the development branch:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released on version [1.5.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.5.0)!

**Comment by igorborgest** [comment]
Hi @ilyanoskov !

It is totally off-topic, but we are stating a [_"Who uses AWS Data Wrangler?"_](https://github.com/awslabs/aws-data-wrangler/tree/dev#who-uses-aws-data-wrangler) section. So feel free to add yourself if you want 😄 .
